pages/ksphere/konvoy/partner-solutions/jenkins/index.md

```diff
@@ -27,7 +27,7 @@ You must have a running Konvoy cluster attached to a Kommander instance. If you
 
 Your Jenkins instance will deploy.
 
-## Installing Jenkins from upstream helm chart
+## Install Jenkins from upstream helm chart
 
```

Comment:
Should probably add a prerequisites section here and mention helm v2 or v3? 
We also need to ensure tiller (in case of helm v2) has the right perms to able to install charts into "jenkins" namespace. Should probably refer to https://v2.helm.sh/docs/rbac/.

Reply:
current chart (version 2.6.4) supports both helm v2 and v3, so updated the docs to reflect that.